test/integration/create-next-app/index.test.ts

```diff
@@ -186,6 +186,36 @@ describe('create next app', () => {
     })
   })
 
+  it('should allow example with GitHub URL with trailing slash', async () => {
+    await usingTempDir(async (cwd) => {
+      const projectName = 'github-app'
+      const res = await run(
+        [
+          projectName,
+          '--example',
+          'https://github.com/vercel/nextjs-portfolio-starter/',
```

Comment:
Both `vercel/next.js` and `vercel/exmaples` are not the "entire repo is a bootstrap template" case, so another vercel example repo `https://github.com/vercel/nextjs-portfolio-starter` is being used for the test.